### ISSUE TITLE
Blocked by cloudflare even after webview verification

### ISSUE THREAD

**Issue by nguyd1** [LNReader/lnreader-sources#669]
### Source information

NovelUpdates

### Steps to reproduce

1. Go to Novel updates source and get a cloud flare message
2. Tap on WebView and pass the cloud flare check
3. Go back to novel updates source and still get a cloud flare message

### Expected behavior

Cloud flare message should be gone

### Actual behavior

Cloud flare message is not gone

### LNReader version

1.1.18

### Android version

Android 13

### Other details

Novelupdates just added cloudflare a few minutes ago it seems like since the chapters were loading normally up until then. I also noticed this issue with any cloudflare source like foxaholic. I have tried a combination of clearing cookies, launching webview, logging in to novelupdates, and killing the app but haven't found any fix to this yet. I also reinstalled the app but see the same behavior.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[1.1.18](https://github.com/LNReader/lnreader/releases/latest)**.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/LNReader/lnreader/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by nguyd1** [comment]
Update: novel updates looks like the website removed the cloudflare I'm able to access the source and chapters again. I'm still having the issue with cloudflare websites though such as taming the villainesses from novelupdates

**Comment by marlau97** [comment]
I still have this issue, tried with and without a VPN, no matter how I try, NovelUpdates is blocked with cloudflare and I can't get it unblocked in the app even if I go to the web view.
